Sources/Sentry/SentryTransaction.m

```diff
@@ -13,14 +13,19 @@
 SentryTransaction ()
 
 /**
- * This transaction span context
+ * This transaction span context.
  */
-@property (nonatomic, strong) SentrySpanContext *spanContext;
+@property (nonatomic) SentrySpanContext *spanContext;
 
 /**
  * A hub this transaction is attached to.
  */
-@property (nonatomic, readonly) SentryHub *_Nullable hub;
+@property (nullable, nonatomic) SentryHub *hub;
```

Comment:
Perfect. The lack of NoOpHub is  the reason I'm using nullable hub.

Reply:
We're not exposing Hub here in any way, usage in cocoa is "only" though SentrySDK, right? So NoOpHub is less important I believe. I added this to .NET because of DI in ASP.NET Core and you can take `IHub` as a dependency. So "DisabledHub" gets injected if Sentry is disabled. But maybe it isn't needed here, specially if null annotation is supported in the language (C# didn't have it at the time).